### PR TITLE
Fetch image frames in msgpack or bytes format

### PR DESCRIPTION
**Pull request by alesgenova**

### CODE REVIEW

File: girder/stemserver_plugin/models/stemimage.py [hunk 1]
```diff
@@ -320,32 +310,40 @@ def _get_vlen_dataset_in_chunks(self, dataset, limit, offset,
...
+                read_size = 1
 
-        if read_size > dataset.shape[0]:
-            read_size = dataset.shape[0]
+            if read_size > dataset.shape[0]:
+                read_size = dataset.shape[0]
+            read_limit = dataset.shape[0]
+        else:
+            read_size = 1
+            read_limit = 1
```
Comment (psavery) [inline girder/stemserver_plugin/models/stemimage.py]: Is the `array_position` supposed to be used as an index somewhere? I don't see it being used other than checking if it is `None`.

**Comment by psavery** [comment]
Unfortunately, I have a feeling that this one is going to clash with #22.

**Review by cjh1 — APPROVED** [review]

**Comment by cjh1** [comment]
@psavery Could you take a look at these changes.

**Review by psavery — APPROVED** [review]
I didn't test it, but it looks good to me.

**Pull request merged**